pkg/controller/status/status_test.go

```diff
@@ -1,89 +1,185 @@
 package status
 
 import (
-	"context"
+	"reflect"
 	"testing"
-
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/klog/v2"
-
-	configv1 "github.com/openshift/api/config/v1"
-	configfake "github.com/openshift/client-go/config/clientset/versioned/fake"
-	"github.com/openshift/insights-operator/pkg/config"
-	"github.com/openshift/insights-operator/pkg/config/configobserver"
-	"github.com/openshift/insights-operator/pkg/utils"
-	kubeclientfake "k8s.io/client-go/kubernetes/fake"
 )
 
-func Test_Status_SaveInitialStart(t *testing.T) {
+func Test_controllerStatus_getStatus(t *testing.T) {
+	type fields struct {
+		statusMap map[string]statusMessage
+	}
+	type args struct {
+		id string
+	}
```

Comment:
Same here the type `fields` is not necessary IMO and the `args` is little bit redundant IMO.